wgengine/router/router_passthrough.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package router
+
+import (
+	"github.com/tailscale/wireguard-go/device"
+	"github.com/tailscale/wireguard-go/tun"
+	"tailscale.com/types/logger"
+)
+
+// SetRoutesFunc applies the given router settings to the OS network stack.
```

Comment:
Mention that this hook itself can be nil, though. 

But I'd rather we ditch this global hook func entirely and make use of it explicit from the one caller.